src/components/Explore/utils.ts

```diff
@@ -396,19 +396,14 @@ function truncateCountyName(countyName: string) {
     : `${countyName.slice(0, 11).trim()}…`;
 }
 
-function getShortLocationLabel(location: Location) {
-  if (location.full_fips_code && location.full_fips_code.startsWith('00')) {
-    return location.state_code;
-  } else {
-    const { county: countyName, state_code } = location;
-    return countyName
-      ? `${truncateCountyName(getAbbreviatedCounty(countyName))}`
-      : state_code.toUpperCase();
-  }
+function getShortLocationLabel(location: Region) {
+  return location.regionType === RegionType.COUNTY
+    ? `${truncateCountyName(location.abbreviation)}`
```

Comment:
we don't need the string interpolation here

```ts
return location.regionType === RegionType.COUNTY
  ? truncateCountyName(location.abbreviation)
  : location.abbreviation
```

Reply:
truu